### PR TITLE
Update uri.py

### PR DESCRIPTION
**Pull request by azagramac**
##### SUMMARY
Create workspaces in Log analytics Azure

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### SUMMARY
Create workspaces in Log analytics Azure

##### ADDITIONAL INFORMATION
Create workspaces in Log analytics Azure.

### CODE REVIEW

**Review by bmillemathias — COMMENTED** [review]
(Comment from someone not owner of the module)

Hi,

Could you change the title to something less generic than "Update uri.py" like "Adding an example for creating workspace on Azure Log Analytics"

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 1]
```diff
@@ -260,6 +260,20 @@
     method: POST
     src: /path/to/my/file.json
     remote_src: yes
+
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]:
```suggestion

```

File: lib/ansible/modules/net_tools/basics/uri.py [hunk 2]
```diff
@@ -260,6 +260,20 @@
...
+- name: Create workspaces in Log analytics Azure
+  uri:
+    url: https://www.mms.microsoft.com/Embedded/Api/ConfigDataSources/LogManagementData/Save
+    method: POST
+    body_format: json
+    status_code: [200, 202]
+    return_content: true
+    headers:
+      Content-Type: application/json
+      x-ms-client-workspace-path: /subscriptions/{{ common.azure.subscription_id }}/resourcegroups/{{ lw_resource_group }}/providers/microsoft.operationalinsights/workspaces/{{ lw_workspaces }}
```
Comment (acozine) [inline lib/ansible/modules/net_tools/basics/uri.py]: This line will need to be 160 characters or under. You can split the line, or change the variable names to shorten them, or use some other method . . . your choice, as long as the line ends up under the limit.

**Review by acozine — COMMENTED** [review]
Thanks @AzagraMac for taking the time to open a PR! This example looks like it could be a useful addition. However, it needs to pass CI before we can merge it. 

The whitespace error is easy to fix - just accept my suggestion.

The line-length error is a little trickier, but you should be able to fix it. Feel free to ask questions in the ansible-docs channel on freenode IRC if you run into problems.

**Pull request merged**